scoring/templates/scoring/base.html

```diff
@@ -47,6 +47,52 @@
         <a class="btn donation ml-2" href="https://www.paypal.com/donate?token=JVAijB-upEdRZMmUvUG04rqaFjj4OfIaEnzewHJZgHGrCCuqOwKlJqaefZttGHMEEo7A45BVdLDBqVkn">Donate</a>
     </nav>
 {% block content %}{% endblock %}
-
+    <footer class="container-fluid bg-dark-grey">
+        <div class="row">
+            <div class="col-12 col-md-4 d-flex flex-column justify-content-between mb-5 mb-md-0">
+                <div class="scorecards-logo"></div>
+                <div class="d-none d-md-flex  flex-row align-items-center mt-4 mt-lg-5">
+                    <a class="logo ceuk-logo" href="https://www.climateemergency.uk/"></a>
+                    <div class="d-flex flex-column align-items-start">
+                        <span>Built by</span>
+                        <a class="logo mysociety-logo" href="https://www.mysociety.org/"></a>
+                    </div>
+                </div>
+            </div>
+            <div class="col-12 col-md-4 mb-5 mb-md-0">
+                <a class="footer-link" href="#">Scorecards</a>
+                <a class="footer-link" href="#">Methodology</a>
+                <a class="footer-link" href="#">How to use the scorecards</a>
+                <a class="footer-link" href="#">About</a>
+                <a class="footer-link" href="#">Contact</a>
+            </div>
+            <div class="col-12 col-md-4">
+                <div>
+                    <div class="newsletter-wrapper">
+                        <h5>Receive the lastes news from Climate Emergency UK</h5>
+                        <p class="newsletter-subtitle">Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint.</p>
+                        <form class="newsletter-form mt-4">
+                            <input class="name" type="text" id="fname" name="firstname" placeholder="Name">
+                            <input class="last-name" type="email" id="email" placeholder="Email" pattern=".+@globex\.com" size="30" required>
```

Comment:
You don’t need a `pattern` attribute here. (Even worse, the current one only allows submission of email addresses that end in "@globex.com"!)

Reply:
To be fair this last one is just for markup and styling purposes, I'll ask CEUK how they manage newsletters, mailchimp or any other platform they use. Then I'll put the correct code.